src/analytics/guaService.ts

```diff
@@ -5,11 +5,13 @@ import { AnalyticsBaseInfo, OperatingSystem } from './analyticsBaseInfo';
  * Google Universal Analytics Service
  */
 export class GUAService {
-    private _visitor: any;
+    private _visitor: ua.Visitor;
     private _getBasePayload: () => any;
 
     constructor(trackingId: string, baseInfo: AnalyticsBaseInfo) {
-        this._visitor = ua(trackingId, baseInfo.clientId, { requestOptions: {}, strictCidFormat: false });
+        const proxy = process.env.HTTP_PROXY || process.env.HTTPS_PROXY;
```

Comment:
why not using proxy-lib: https://github.com/Icenium/proxy-lib#getproxysettings ?
In case you have a proxy, you must set it to CLI. But in case you want to use VSCode, with the current implementation you'll have to set it to CLI and in the env variable as well.

Reply:
The current implementation is just for testing. Without this change it's would be very hard for the QAs to verify this commit.